File: c programming/basic/grade.c
#include<stdio.h>
int main()
{
    int marks;
    printf("enter marks: ");
    scanf("%d",&marks);
    if(marks<30 && marks>0)
    {
        printf("the grade is c");
    }
    else if (marks>30 && marks<70)
    {
        printf("the grade is B");
    }
    else if(marks>70 && marks<90)
    {
        printf("the grade is A");
    }
    else if (marks >90 && marks <100)
    {
        printf("the grade is A+");
    }
    else("marks is invalid");
    return 0;
}

File: jAVA/if_else/grade.java
package if_else;


import java.util.Scanner;
public class grade {
    public static void main(String[] args) {
        Scanner Sc= new Scanner(System.in);
        System.out.print("Enter marks:");
        int n= Sc.nextInt();
        if(n>80 && n<=100)
        System.out.print("very good");
        else if(n>60)
        System.out.print("Good");
        else if (n>40)
        System.out.print("Average");
        else
        System.out.print("fail");
        Sc.close();
        }
}
